packages/@lwc/rollup-plugin/src/index.ts

```diff
@@ -7,11 +7,13 @@
 import fs from 'fs';
 import path from 'path';
 import { URLSearchParams } from 'url';
+import { XMLParser } from 'fast-xml-parser';
```

Comment:
I don't have a strong opinion about XML parsers; I kind of just grabbed one at random.

I figure it's better to have something battle-tested than something fast – there could be a lot of XML files out there, and we want to be able to handle all of them.